modules/nw-udn-limitations.adoc

```diff
@@ -9,7 +9,9 @@
 While user-defined networks (UDN) offer highly customizable network configuration options, there are limitations that cluster administrators and developers should be aware of when implementing and managing these networks. Consider the following limitations before implementing a user-defined network.
 
 //Check on the removal of the DNS limitation for 4.18 or 4.17.z.
-* *DNS limitations*: DNS lookups for pods resolve to the pod's IP address on the cluster default network. Even if a pod is part of a user-defined network, DNS lookups will not resolve to the pod's IP address on that user-defined network.
+* *DNS limitations*:
+** DNS lookups for pods resolve to the pod's IP address on the cluster default network. Even if a pod is part of a user-defined network, DNS lookups will not resolve to the pod's IP address on that user-defined network. However, DNS lookups for services and external entities will function as expected.
```

Comment:
Existing content, but can future tense be worked around here? 

```suggestion
** DNS lookups for pods resolve to the pod's IP address on the cluster default network. Even if a pod is part of a user-defined network, DNS lookups cannot resolve to the pod's IP address on that user-defined network. However, DNS lookups for services and external entities function as expected.
```

or

```suggestion
** DNS lookups for pods resolve to the pod's IP address on the cluster default network. Even if a pod is part of a user-defined network, DNS lookups do not resolve to the pod's IP address on that user-defined network. However, DNS lookups for services and external entities function as expected.
```

If either messes up the semantics, feel free to ignore :)